src/database/models/users.js

```diff
@@ -2,12 +2,29 @@ module.exports = (sequelize, DataTypes) => {
   const Users = sequelize.define(
     'Users',
     {
-      first_name: DataTypes.STRING,
-      last_name: DataTypes.STRING,
-      email: DataTypes.STRING,
-      password: DataTypes.STRING,
-      role: DataTypes.STRING,
-      gender: DataTypes.STRING
+      firstName: {
+        allowNull: false,
+        type: DataTypes.STRING
+      },
+      lastName: {
+        allowNull: false,
+        type: DataTypes.STRING
+      },
+      email: {
+        allowNull: false,
+        unique: true,
+        type: DataTypes.STRING
```

Comment:
you can as well add the property `isEmail: true` to allow checking for email formats

Reply:
Thanks for your feedback. I did consider that but I decided to leave it out since user input must pass through the validation middlewares.